src/Imath/ImathExport.h

```diff
@@ -3,15 +3,58 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+// See the document in OpenEXR docs/SymbolVisibility.md for background
+// on all these types
 #if defined(IMATH_DLL)
```

Comment:
Could you add a comment to each of the blocks here indicating what machine/os the associated settings presumably apply to? That would make it a bit easier to read. Presumably you explain that in docs/SymbolVisibility.md, but sprinkling some of that explanation here would be helpful.